terraform/Azure/only_private_blob_storage/only_private_blob_storage_plan_mock_2.json

```diff
@@ -0,0 +1,536 @@
+{
```

Comment:
You split the mocks into different files.. I liked it! 😁
We may need to do that in the AWS policies as well..